src/main/webapp/script.js

```diff
@@ -1,15 +1,32 @@
+var counter = 0;
 function onSignIn(googleUser) {
   // Useful data for your client-side scripts:
   var profile = googleUser.getBasicProfile();
-  console.log("ID: " + profile.getId()); // Don't send this directly to your server!
+  console.log('ID: ' + profile.getId()); // Don't send this directly to your server!
   console.log('Full Name: ' + profile.getName());
   console.log('Given Name: ' + profile.getGivenName());
   console.log('Family Name: ' + profile.getFamilyName());
-  console.log("Image URL: " + profile.getImageUrl());
-  console.log("Email: " + profile.getEmail());
+  console.log('Image URL: ' + profile.getImageUrl());
+  console.log('Email: ' + profile.getEmail());
 
   // The ID token you need to pass to your backend:
   var id_token = googleUser.getAuthResponse().id_token;
-  console.log("ID Token: " + id_token);
-  window.location.replace("dashboard.html");
+  console.log('ID Token: ' + id_token);
+
+  document.getElementById('user-profile').innerHTML = profile.getGivenName();
+  document.getElementById('headert').innerHTML =
+    profile.getGivenName() + ' ' + profile.getFamilyName();
+  document.getElementById('email').innerHTML = profile.getEmail();
+}
+
+function Switch() {
```

Comment:
[Method names are written in lowerCamelCase](https://google.github.io/styleguide/jsguide.html#naming-method-names)

Reply:
Also, I recommend a more descriptive name. E.g. `toggleContent`

Reply:
Thanks, I'll fix it